dotnet-build.sh

```diff
@@ -1,16 +1,42 @@
 #!/bin/bash
+
 # return failing exit code if any command fails
 set -e
 
 # enable nullglob - allows filename patterns which match no files to expand to a null string, rather than themselves
 shopt -s nullglob
 
+# parse flags
+while getopts ":vt" opt; do
+  case $opt in
+    v) VERBOSE=true;;
+    t) WITH_TESTING=true;;
+  esac
+done
+
+# show input environment args
+if [ $VERBOSE ]
+then
+    echo "env: BITBUCKET_CLONE_DIR=$BITBUCKET_CLONE_DIR"
+    echo "env: MYGET_ACCESS_TOKEN=$MYGET_ACCESS_TOKEN" | sed -e 's/=.\+/=******/g'
+    echo "env: PROJECT_NAME=$PROJECT_NAME"
+    echo "env: PROJECT_FILE=$PROJECT_FILE"
+    echo "env: WITH_TESTING=$WITH_TESTING"
+    echo ""
+fi
+
 ROOTDIRECTORY="/api"
 
 # go to the workdir
 if [ -n "$BITBUCKET_CLONE_DIR" ]
 then
-    cd "$BITBUCKET_CLONE_DIR" || return
+    # support for relative path
+    case $BITBUCKET_CLONE_DIR in
```

Comment:
This is dangerous.
If both root and "user" have the same clone dir and it **is** relative then this will always attempt to run on the directory of root.
If the directory is relative without this check it will always run the correct version (e.g the user version)


What are you trying to solve with this check?

Reply:
It was only a suggestion to be discussed further comming week:-) this was also for local dev / debugging

Reply:
It's all good! Feedback is always meant to be constructive!